ConsoleUI.js

```diff
@@ -81,6 +81,30 @@ const consoleUI = (function (readline) {
       console.log(
         'GET OUT OF JAIL FREE: Card was used and returned to the discard deck.'
       ),
+    auctionInstructions: () =>
+      console.log(
+        'AUCTION: The rules are as follows. All players who can afford the property will continue to bid until there is one clear winner. Each player will get a chance to input their bid for the round. If a player enters nothing or a bid below the current bidding cost, that player is out of the auction. At least one player must bid to exit the auction.'
```

Comment:
`...rules are as follows:` <- colon (`:`) instead of full stop (`.`).
`All players who can afford the property will bid in rounds until there is one clear winner.` <- Add `rounds` here as this sentence appears to be the high-level summary.

`Each player...` -> `Players take turns inputting their bids each round.` ? Tbh, not sure if we need this sentence; I think reading the sentence before and after without this still makes sense ?

`If a player bids nothing or a bid below..` <- Switch `enters nothing` to `bids nothing` for conciseness.

Do we want to add that bit about "If no one can bid there is no auction" or is that handled at the time of request ? This appears to be after the auction begins, meaning that the verification of eligible auction folk has been done

Reply:
That will be done outside. :)